CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Generate a bmap file for the resin-sdcard image [Michal]
```

Comment:
same comment here about whether to use the naming "resinos-img" rather than "resin-sdcard"